sbin/unit-tests

```diff
@@ -252,7 +330,7 @@ run_all_tests() {
     # Run C++ tests
     run_cpp_tests
 
-    # Run C coordinator tests
```

Comment:
Not only C? Let's at least group with the C++ step later if we make this one generic